lib/commons/builder/queries/legislative_index_terms.rq.liquid

```diff
@@ -15,6 +20,11 @@ WHERE {
   OPTIONAL { ?term (wdt:P582|wdt:P576) ?termEnd. }
   OPTIONAL { ?term (wdt:P155|wdt:P1365) ?termReplaces }
   OPTIONAL { ?term (wdt:P156|wdt:P1366) ?termReplacedBy }
+  OPTIONAL {
+    ?termSpecificPosition wdt:P31 wd:Q4164871 ;
```

Comment:
Are these always going to be directly an `instance of` `position`? I wonder if it might be good future-proofing to make this `P31/P279*`...

Reply:
I'd assumed position was one of those things that had to be direct, but given public office and things like [this chair](https://www.wikidata.org/wiki/Q81834) I think you're right. Now changed :-)